src/solvers.jl

```diff
@@ -57,17 +57,23 @@ problems must have unique node ids.
 function get_field_assembly(solver::Solver)
     problems = get_field_problems(solver)
 
-    M = SparseMatrixCOO()
-    K = SparseMatrixCOO()
-    Kg = SparseMatrixCOO()
-    f = SparseMatrixCOO()
-    fg = SparseMatrixCOO()
-
-    for problem in problems
+    problem = problems[1]
+    M = problem.assembly.M
+    K = problem.assembly.K
+    f = problem.assembly.f
+    K_csc = problem.assembly.K_csc
```

Comment:
Here, we should not have several different presentations of stiffness matrix, but instead, have only one and define necessary functions so that code works for both of types?